crates/core_simd/src/masks.rs

```diff
@@ -1,3 +1,13 @@
+/// The error type returned when converting an integer to a mask fails.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct TryFromMaskError(());
```

Comment:
should this carry the bad value as a payload and then you can give the bad value as part of the `Display` impl?

Reply:
I copied `TryFromIntError` for this, which doesn't hold the value.  Since the types are `Copy` I'm guessing it's easy enough for the user to do that if they want it.